tests/test_telemetry_monitor.py

```diff
@@ -520,7 +520,7 @@ def test_telemetry_serial_models(fileutils, test_dir, wlmutils, monkeypatch):
         exp = Experiment(exp_name, launcher=test_launcher, exp_path=test_dir)
 
         # create run settings
-        app_settings = exp.create_run_settings("python", test_script)
+        app_settings = exp.create_run_settings(sys.executable, test_script)
```

Comment:
Thanks for fixing these by the way! Should help limit the number of false negatives!!